strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/io/ArtifactFile.java

```diff
@@ -1,50 +0,0 @@
-package org.carlspring.strongbox.io;
-
-import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
-
-import java.io.File;
-
-/**
- * This implementation contains additional information about Artifact File itself, such as {@link Repository} and
- * {@link ArtifactCoordinates}. <br>
- * Note that this is only about "File System" (Unix or Windows) as common {@link File}.
- * 
- * @author mtodorov
- * 
- */
-public class ArtifactFile
```

Comment:
@fuss86 you are cool! I did not dare to do it in my time, but always wanted :)

Reply:
Good job! :)

Reply:
:)